integration_tests/projects/004_script_paths/dbt_project.yml

```diff
@@ -0,0 +1,11 @@
+name: "fal_test"
+version: "1.0.0"
+config-version: 2
+profile: "fal_test"
+source-paths: ["models"]
+data-paths: ["data"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+vars:
+  scripts-path: "scripts"
```

Comment:
I am guessing setting it top-level was not permitted? Should we prefix it with `fal`: `fal-scripts-path`?